Isosurfacing_3/examples/Isosurfacing_3/all_cartesian_cube.cpp

```diff
@@ -0,0 +1,82 @@
+#include <CGAL/Cartesian_grid_3.h>
+#include <CGAL/Dual_contouring_3.h>
+#include <CGAL/Explicit_cartesian_grid_domain.h>
+#include <CGAL/Marching_cubes_3.h>
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/boost/graph/IO/OFF.h>
+
+typedef CGAL::Simple_cartesian<double> Kernel;
+typedef typename Kernel::FT FT;
+typedef typename Kernel::Point_3 Point;
+typedef typename Kernel::Vector_3 Vector;
+
+typedef CGAL::Cartesian_grid_3<Kernel> Grid;
+
+typedef std::vector<Point> Point_range;
```

Comment:
Maybe use a `std::deque`.

Reply:
For what reason? Would you expect better insertion performance because of the avoided copies when reallocating a vector over the higher number of allocations of fixed-size blocks in the deque?